src/Table/Column.php

```diff
@@ -359,7 +359,7 @@ public function getHeaderCellHtml(Field $field = null, $value = null)
      */
     public function getTotalsCellHtml(Field $field, $value)
     {
-        return $this->getTag('foot', $this->getApp()->ui_persistence->typecastSaveField($field, $value));
+        return $this->getTag('foot', (string) $this->getApp()->ui_persistence->typecastSaveField($field, $value));
```

Comment:
what is the definition of direction of `typecastSaveField`? (from ui OR to ui?)

Reply:
because `ui_persistence` should probably never cast to anything else than string when casting "to ui"/"from php"

Reply:
thats a good thought you have there. I think Persistence/UI is one of the things that could need some rework or even be dropped completely. At least the name is very misleading as AFAIK it does not persist anything but makes data usable for UI or casts posted form data back.

Reply:
The only thing I care is "to ui"/"from php" cast method where this issue should be fixed. So find it and cast there.

Reply:
I fixed the code on right place. Not tested, fix failing cases if there are any.